src/parser.ts

```diff
@@ -4140,7 +4140,7 @@ export class Parser extends DiagnosticEmitter {
           break;
         }
         // InstanceOfExpression
-        case Token.INSTANCEOF: {
+        case Token.InstanceOf: {
```

Comment:
This one is somewhat similar, in that the operation here is `instanceof`, so perhaps `Instanceof` to indicate?

Reply:
Otoh, I see now that this notation becomes weird on its own on `OpenBracket` and such. Hmm.

Reply:
I switched to `Instanceof`